<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSettingNetForm</class>
 <widget class="QWidget" name="QSettingNetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="ed_settingnet_ip">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ed_settingnet_mask">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ed_settingnet_getway">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>140</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ed_settingnet_dns">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>220</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="le_settingnet_ip">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="le_settingnet_mask">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="le_settingnet_getway">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="le_settingnet_dns">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>290</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_zhixing">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>320</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="le_wifiname">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>520</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_netset">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="HNWIFIWidget" name="m_pwifi">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HNWIFIWidget</class>
   <extends>QTableView</extends>
   <header>hnwifiwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
